BPRBlazor/Components/Common/LoadingIndicator.razor.css

```diff
@@ -0,0 +1,5 @@
+﻿.center {
+    position: absolute;
+    top: 47.5vh;
+    left: 52.5vw;
```

Comment:
Too specific? or am I not seeing it? we'll leave it for the big redesign

Reply:
Made them both 50 now so it's completely centered on the screen rather than centered on <main></main>.
Still not very happy with the way I've done this, just not sure how else to do it 😞 Tried with a modal and stuff but couldn't get it to look nice...